src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,23 @@
+
+.TodoInfo {
+  width: 200px;
+  padding: 8px;
+  margin: 12px 0;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: antiquewhite;
```

Comment:
nit: Consider using a more descriptive name for the background color, or using a variable for the color. Example: $bg-color: #f9e9d9; background-color: $bg-color;